osCam/core/urls.py

```diff
@@ -8,4 +8,5 @@
 urlpatterns = [
     path('', views.home, name='home'),    
     path('feed/', views.feed, name='feed'),
+    # path('/settings/', views.settings, name='settings'),
```

Comment:
any particular reason for the commented path?

Reply:
nope. probably don't need it. i think this was later changed to config and I'll have to ask kevin abt URL mapping.

Reply:
@DJ-IRL I'm sure you already will but please make sure to take out unnecessary comments. Comments are great if they provide context.

Reply:
yeah this was left for reference for when we talk about URLs, as this was my first intuition on how to implement it but i later saw how it was already configured and adapted to that. 

shoulda just made note of it somewhere else